### ISSUE TITLE
Freezing after VERY specific bash command is run. How to debug?

### ISSUE THREAD

**Issue by donaastor** [alacritty/alacritty#6979]
After I `cd` to a custom `tmpfs` and run `ls -la` and open Geany alongsize Alacritty, Alacritty fails to redraw when I resize Geany... YEAH it is VERY specific. Even the logs aren't useful. There have been way more resize events (they should have happened). Basically, Alacritty stops refreshing completely. After I swap desktops the terminal is gone. Just nothing is drawn. How do I even debug this? My programming experience is hurting very much from this.

### System

OS: ArchLinux, X11, i3 - everyting is up to date

### Logs

```
Created log file at "/tmp/Alacritty-111399.log"
[0.000003140s] [INFO ] [alacritty] Welcome to Alacritty
[0.000028500s] [INFO ] [alacritty] Version 0.12.1 (5fdfd47f)
[0.000033390s] [INFO ] [alacritty] Running on X11
[0.000129370s] [INFO ] [alacritty] Configuration files loaded from:
                                     "/home/***/.config/alacritty/alacritty.yml"
[0.002040292s] [INFO ] [alacritty] Using GLX 1.4
[0.038294309s] [INFO ] [alacritty] Window scale factor: 2
[0.039053196s] [DEBUG] [alacritty] Loading "monospace" font
[0.045612020s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: TARGET_LIGHT, render_mode: "Lcd", lcd_filter: 1 }
[0.048537198s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Bold, load_flags: TARGET_LIGHT, render_mode: "Lcd", lcd_filter: 1 }
[0.050844679s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Italic, load_flags: TARGET_LIGHT, render_mode: "Lcd", lcd_filter: 1 }
[0.053253840s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Bold Italic, load_flags: TARGET_LIGHT, render_mode: "Lcd", lcd_filter: 1 }
[0.056325318s] [INFO ] [alacritty] Running on AMD Radeon RX 6650 XT (navi23, LLVM 15.0.7, DRM 3.52, 6.3.5-arch1-1)
[0.056334548s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.059883163s] [DEBUG] [alacritty] Filling glyph cache with common glyphs
[0.065640301s] [INFO ] [alacritty] Cell size: 18 x 34
[0.065650731s] [INFO ] [alacritty] Padding: 0 x 0
[0.065654971s] [INFO ] [alacritty] Width: 800, Height: 600
[0.085702571s] [INFO ] [alacritty] PTY dimensions: 17 x 44
[0.087344585s] [INFO ] [alacritty] Initialisation complete
[0.087677664s] [DEBUG] [alacritty_terminal] New num_cols is 106 and num_lines is 62
[0.088360261s] [INFO ] [alacritty] Padding: 0 x 0
[0.088366621s] [INFO ] [alacritty] Width: 1920, Height: 2111
[4.465499254s] [DEBUG] [alacritty_terminal] New num_cols is 213 and num_lines is 62
[4.469340340s] [INFO ] [alacritty] Padding: 0 x 0
[4.469348359s] [INFO ] [alacritty] Width: 3840, Height: 2111
[12.900269800s] [DEBUG] [alacritty_terminal] New num_cols is 106 and num_lines is 62
[12.900432119s] [INFO ] [alacritty] Padding: 0 x 0
[12.900438879s] [INFO ] [alacritty] Width: 1920, Height: 2111
[21.279172428s] [DEBUG] [alacritty_terminal] New num_cols is 110 and num_lines is 62
[21.279291727s] [INFO ] [alacritty] Padding: 0 x 0
[21.279298417s] [INFO ] [alacritty] Width: 1997, Height: 2111
^C
```

**Comment by kchibisov** [comment]
What exactly fails to redraw? Is it still getting the events when running with `--print-events` or you don't see anything special in the output?

**Comment by donaastor** [comment]
> What exactly fails to redraw?

I will provide a whole video, I think it's easier that way.
> Is it still getting the events when running with `--print-events` or you don't see anything special in the output?

I will try that too

**Comment by donaastor** [comment]
Watch my video at https://github.com/donaastor/debugging_videos

**Comment by kchibisov** [comment]
The only thing that matters is whether you get `--print-events` while the bug happening or not. From the video it's very hard to understand, but given that resizing bugs, your issue might be fixed with https://github.com/alacritty/alacritty/pull/6958 patches.

**Comment by donaastor** [comment]
No, I don't get any events in the log.

**Comment by chrisduerr** [comment]
Can you try the latest master?

**Comment by donaastor** [comment]
@chrisduerr I will tomorrow.

**Comment by kchibisov** [comment]
I don't think latest master will do anything, only the branch I've linked.

**Comment by kchibisov** [comment]
Closed by #7006
